test.sh

```diff
@@ -0,0 +1 @@
+cargo build --all && cargo test --all
```

Comment:
Need a shebang for portability

```suggestion
#!/usr/bin/env bash
cargo build --all && cargo test --all
```

Reply:
Thanks! ac28df1